SQL/paradise_schema.sql

```diff
@@ -367,17 +367,22 @@ DROP TABLE IF EXISTS `library`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `library` (
-  `id` int(11) NOT NULL AUTO_INCREMENT,
-  `author` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
-  `title` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
-  `content` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
-  `category` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
-  `ckey` varchar(32) COLLATE utf8mb4_unicode_ci NOT NULL,
-  `flagged` int(11) NOT NULL,
-  PRIMARY KEY (`id`),
-  KEY `ckey` (`ckey`),
-  KEY `flagged` (`flagged`)
-) ENGINE=InnoDB AUTO_INCREMENT=4537 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+	`id` INT(11) NOT NULL AUTO_INCREMENT,
+	`author` MEDIUMTEXT NOT NULL COLLATE 'utf8mb4_unicode_ci',
+	`title` MEDIUMTEXT NOT NULL COLLATE 'utf8mb4_unicode_ci',
+	`content` MEDIUMTEXT NOT NULL COLLATE 'utf8mb4_unicode_ci',
+	`ckey` VARCHAR(32) NULL DEFAULT '' COLLATE 'utf8mb4_unicode_ci',
+	`reports` MEDIUMTEXT NOT NULL COLLATE 'utf8mb3_general_ci',
+	`summary` MEDIUMTEXT NOT NULL COLLATE 'utf8mb3_general_ci',
+	`ratings` DOUBLE NULL DEFAULT '0',
+	`raters` MEDIUMTEXT NOT NULL COLLATE 'utf8mb3_general_ci',
+	`primaryCategory` INT(11) NULL DEFAULT '0',
+	`secondaryCategory` INT(11) NOT NULL DEFAULT '0',
+	`tertiaryCategory` INT(11) NULL DEFAULT '0',
```

Comment:
We tend to `snake_case` tables, as most SQL clients are case-agnostic. Don't have to change it but would keep consistency with most the tables (please ignore the outliers).

Reply:
done